app/Models/Order.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Models;
+
+use App\Traits\SortableTrait;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Order extends Model
+{
+    use HasFactory, SortableTrait;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<int, string>
+     */
+    protected $fillable = [
+        'user_id',
+        'client_id',
+        'status',
+        'invoice_num',
+        'total_quantity',
+        'total_price'
+    ];
+
+    public function scopeSearch($query, $searchTerm)
```

Comment:
Jeśli dobrze, rozumiem to ta funkcja ma znaleźć zamówienia, po relacji Order -> client oraz Order->user.
Można to osiągnąć nie korzystając z raw queries, tylko z relacji i 'whereHas'. Poniżej przykład znaleziony w necie

```php
Categories::whereHas('products', function ($query) use ($searchString){
        $query->where('name', 'like', '%'.$searchString.'%');
    })
```